save-common/src/commonMain/kotlin/org/cqfn/save/core/logging/Logger.kt

```diff
@@ -16,13 +18,24 @@ import kotlinx.datetime.toLocalDateTime
 /**
  *  Logging mode
  */
-var logType: LogType = LogType.WARN
+expect var logType: GenericAtomicReference<LogType>
 
 /**
  * Whether to add time stamps to log messages
  */
 var isTimeStampsEnabled: Boolean = false
 
+/**
+ *  Class that holds value and shares atomic reference to the value (native only)
+ */
+@Suppress("USE_DATA_CLASS")
+expect class GenericAtomicReference<T> {
```

Comment:
Should be moved from `Logger.kt` to something more appropriate, like where we have `AtomicBoolean`

Reply:
moved to `PlatformUtils.kt`